cmd/log.go

```diff
@@ -41,12 +45,13 @@ var logCmd = &cobra.Command{
 		// Retrieve and set the currentComponent
 		currentComponent := getComponent(client, argComponent, currentApplication, currentProject)
 
-		err = component.GetLogs(client, currentComponent, stdout)
+		// Retrieve the log
+		err = component.GetLogs(client, currentComponent, logWatch, stdout)
 		checkError(err, "Unable to retrieve logs, does your component exist?")
 	},
 }
 
 func init() {
 	rootCmd.AddCommand(logCmd)
-
+	logCmd.Flags().BoolVarP(&logWatch, "watch", "w", false, "Stream logs")
```

Comment:
can we use `--follow` / `-f`? 
I think that `-f` is more standard for this kind of things.  `kubect logs -f`, `tail -f` ...